examples/notebooks/stationarity_detrending_0.ipynb

```diff
@@ -0,0 +1,304 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from statsmodels.tsa.stattools import kpss\n",
+    "def kpss_test(timeseries):\n",
+    "    print ('Results of KPSS Test:')\n",
+    "    kpsstest = kpss(timeseries, regression='c', nlags=None)\n",
+    "    kpss_output = pd.Series(kpsstest[0:3], index=['Test Statistic','p-value','Lags Used'])\n",
+    "    for key,value in kpsstest[3].items():\n",
+    "        kpss_output['Critical Value (%s)'%key] = value\n",
+    "    print (kpss_output)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The ADF tests gives the following results – test statistic, p value and the critical value at 1%, 5% , and 10% confidence intervals.\n",
+    "\n",
+    "ADF test is now applied on the data."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "adf_test(sunspots['SUNACTIVITY'])"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Critical value of p is assumed to be 0.05. Based upon the p-value of ADF test, the null hypothesis can not be rejected. Hence, the series is non-stationary."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The KPSS tests gives the following results – test statistic, p value and the critical value at 1%, 5% , and 10% confidence intervals.\n",
+    "\n",
+    "KPSS test is now applied on the data."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "kpss_test(sunspots['SUNACTIVITY'])"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Critical value of p is assumed to be 0.05. Based upon the p-value, the null hypothesis can not be rejected. Hence, the series is stationary as per the KPSS test. \n",
+    "\n",
+    "It is always better to apply both the tests, so that it can be ensured that the series is truly stationary. Possible outcomes of applying these stationary tests are as follows:\n",
+    "\n",
+    "Case 1: Both tests conclude that the series is not stationary - The series is not stationary  \n",
+    "Case 2: Both tests conclude that the series is stationary - The series is stationary  \n",
+    "Case 3: KPSS indicates stationarity and ADF indicates non-stationarity - The series is trend stationary. Trend needs to be removed to make series strict stationary  \n",
+    "Case 4: KPSS indicates non-stationarity and ADF indicates stationarity - The series is difference stationary. Differencing is to be used to make series stationary  \n",
+    "\n",
+    "Here, due to the differcne in the results from ADF test and KPSS test, it can be inferred that the series is trend stationary and not strict stationary. The series can be detrended by differencing or by model fitting."
```

Comment:
Do you have a citation for this? Often I've heard that if KPSS and ADF disagree then there is not enough information to determine stationarity.

Also, typo with "differcne"

Reply:
This is based upon the following understanding which I have developed from various sources:
KPSS test:
Null Hypothesis: the process is trend-stationary
Alternative Hypothesis: the process has a unit root 
ADF test:
Null Hypothesis: the process has a unit-root ("difference stationary")
Alternative Hypothesis: the process has no unit root.

I inferred Case 3 and Case 4 based upon the above understanding. In a trend stationary series, trend needs to be removed and the detrended series needs to be checked for stationarity. Similarly, the difference stationary series requires differencing and then, checked for stationarity.  

I await your feedback or correction on my understanding.